scrapy/pipelines/files.py

```diff
@@ -9,12 +9,13 @@
 import mimetypes
 import os
 import time
+from os import PathLike
```

Comment:
Note that the latest `master` has introduced `isort` and CI will fail due to this line. See https://docs.scrapy.org/en/latest/contributing.html#pre-commit, it will fix this when you merge or rebase from the latest `master`.

Reply:
Got it. Thanks